src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/XamarinProject.cs

```diff
@@ -415,12 +414,6 @@ protected void AddNuGetConfigSources (string nugetConfigPath)
 				ExtraNuGetConfigSources = new List<string> ();
 			}
 
-			if (TargetFramework?.IndexOf ("net8.0", StringComparison.OrdinalIgnoreCase) != -1
-				|| TargetFrameworks?.IndexOf ("net8.0", StringComparison.OrdinalIgnoreCase) != -1) {
-				ExtraNuGetConfigSources.Add ("https://api.nuget.org/v3/index.json");
```

Comment:
Removing this one has caused a test failures, but I'm working on getting the required packages mirrored to `dotnet-public`. That is probably a better path anyway.